### DISCUSSION TITLE
How can I organize the web with Falcon?

### DISCUSSION THREAD

**Discussion by gusxor** [phalcon/cphalcon#16470] [Q&A]
Everyone, please help me 
It's a project that uses Falcon as a school project.
However, I don't think I installed it properly, so it's different from what I found without eating the Falcon command. I'd like to know how to install it from the beginning. 
You can tell me just one of the centeros and Ubuntu. Please
I would like to use only putty and xshell, which are ssh clients, to configure the web page and upload the Falcon framework on the php. Please help me
I'm trying to open the web using aws EC2.'

**Comment by corentin-begne** [comment]
Use pecl to install it, it's easier and you can choose the version that you want :  
https://pecl.php.net/package/phalcon  
Don't forget to install psr and init it before Phalcon.  
https://phalcon.io/en-us/download/linux

**Comment by kgrammer** [comment]
This is an older thread, but for clarification, Phalcon v5 no longer requires PSR.
